packages/shared/src/components/cards/PostList.tsx

```diff
@@ -39,54 +39,52 @@ export const PostList = forwardRef(function PostList(
   const onPostCardClick = () => onPostClick(post);
   const { trending } = post;
 
-  const card = (
-    <ListCard
-      {...props}
-      className={getPostClassNames(post, className)}
-      ref={ref}
+  return (
+    <ConditionalWrapper
+      condition={!!trending}
+      wrapper={(component) => getTrendingFlag(component, trending, true)}
     >
```

Comment:
I'm not really sure this is the right way.
Multiple places doing the same thing, is there not a way we can uniform a cards raisedLabel based on props?

if it has `pinnedAt` show pinned, if it has trending -> show that, else nothing?
Keep in mind any post can be pinned.

Reply:
Let me check how we can uniform it.